src/presentation/cli/controller/runtime.go

```diff
@@ -0,0 +1,256 @@
+		Short: "GetPhpConfigs",
+		Run: func(cmd *cobra.Command, args []string) {
+			hostname, err := getHostname(hostnameStr)
+			if err != nil {
+				cliHelper.ResponseWrapper(false, err.Error())
+			}
+
+			requestBody := map[string]interface{}{
+				"hostname": hostname.String(),
+			}
+
+			cliHelper.ServiceResponseWrapper(
+				controller.runtimeService.ReadPhpConfigs(requestBody),
+			)
+		},
+	}
+
+	cmd.Flags().StringVarP(&hostnameStr, "hostname", "n", "", "Hostname")
+	return cmd
+}
+
+func parsePhpModules(rawPhpModules []string) []entity.PhpModule {
+	modules := []entity.PhpModule{}
+	for _, rawModule := range rawPhpModules {
+		rawModuleParts := strings.Split(rawModule, ":")
+		rawModulePartsLength := len(rawModuleParts)
+		if rawModulePartsLength == 0 {
+			slog.Debug("PhpModuleEmpty", slog.String("module", rawModule))
+			continue
+		}
+
+		moduleName, err := valueObject.NewPhpModuleName(rawModuleParts[0])
+		if err != nil {
+			slog.Debug("InvalidPhpModuleName", slog.Any("name", rawModuleParts[0]))
+			continue
+		}
+
+		moduleStatus := true
+		if rawModulePartsLength > 1 {
+			var err error
+			moduleStatus, err = sharedHelper.ParseBoolParam(rawModuleParts[1])
+			if err != nil {
+				moduleStatus = false
+			}
+		}
+
+		modules = append(modules, entity.NewPhpModule(moduleName, moduleStatus))
+	}
+
+	return modules
+}
+
+func parsePhpSettings(rawPhpSettings []string) []entity.PhpSetting {
+	settings := []entity.PhpSetting{}
+	for _, rawSetting := range rawPhpSettings {
+		rawSettingParts := strings.Split(rawSetting, ":")
+		if len(rawSettingParts) == 0 {
+			slog.Debug("PhpSettingEmpty", slog.String("setting", rawSetting))
+			continue
+		}
+
+		settingName, err := valueObject.NewPhpSettingName(rawSettingParts[0])
+		if err != nil {
+			slog.Debug(
+				"InvalidPhpSettingName", slog.Any("name", rawSettingParts[0]),
+			)
+			continue
+		}
+
+		settingValue, err := valueObject.NewPhpSettingValue(rawSettingParts[1])
+		if err != nil {
+			slog.Debug(
+				"InvalidPhpSettingValue", slog.Any("value", rawSettingParts[1]),
+			)
+			continue
+		}
+
+		settings = append(
+			settings,
+			entity.NewPhpSetting(
+				settingName, settingValue, []valueObject.PhpSettingOption{},
+			),
+		)
+	}
+
+	return settings
+}
+
+func (controller *RuntimeController) UpdatePhpConfig() *cobra.Command {
+	var hostnameStr, phpVersionStr string
+	var modulesSlice, settingsSlice []string
+
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "UpdatePhpConfigs",
+		Run: func(cmd *cobra.Command, args []string) {
+			hostname, err := getHostname(hostnameStr)
+			if err != nil {
+				cliHelper.ResponseWrapper(false, err.Error())
+			}
+
+			requestBody := map[string]interface{}{
+				"hostname": hostname.String(),
+				"version":  phpVersionStr,
+			}
+
+			if len(modulesSlice) > 0 {
+				requestBody["modules"] = parsePhpModules(modulesSlice)
+			}
+
+			if len(settingsSlice) > 0 {
+				requestBody["settings"] = parsePhpSettings(settingsSlice)
+			}
```

Comment:
This ain't wrong, however, I instructed you to read how it was done on Control. On Control we have a FromString constructor which brings these logics here into the VO so the domain can control the serialization and deseralization of the data. You don't need to change, but it would be better to have the FromString constructor to make it easier to maintain the process.

Reply:
Done!